javascripts/CarLot.js

```diff
@@ -0,0 +1,40 @@
+// main IIFE function
+//variables
+// var carContainer = document.getElementsByClassName("container");
+var CarLot = (function () {
+//The first IIFE should add a public function that loads the inventory.
+//json file and stores the inventory in a private variable.
+// It should also expose a public getter to read the array of cars (e.g. getInventory).
+  	return {
+
+  		loadInventory : function(){
+	  		var inventory = [];
+	  		var inventoryLoader = new XMLHttpRequest();
+		  		inventoryLoader.addEventListener ("load",CarLot.executeThisCodeAfterFileLoaded);
+				inventoryLoader.addEventListener("error",CarLot.executeThisCodeWhenFailure)
+				inventoryLoader.open ("GET", "inventory.json");
+				inventoryLoader.send();
+		},
+
+  		executeThisCodeAfterFileLoaded : function(){
+    		var data = JSON.parse(this.responseText);
+    		console.log("my Data is : ", data);
+    		// CarLot.resetBoarderBackground();
+    		CarLot.makeDom(data);
```

Comment:
you are calling makeDom which is a function that exists in another file.  This file is the first that is loaded.  That means the augmentor is not loaded yet. This is causing your click event errors.  you need to change the order of your script tags